download/index.php

```diff
@@ -16,6 +16,19 @@
     $filter[] = 'dev_id IN ('.$qm.')';
     $params = array_merge($params, $regs);
 }
+
+if (!empty($_GET['from_id'])) {
+    $fromid = $_GET['from_id'];
+    if (!empty($_GET['till_id'])) {
+	    $tillid = $_GET['till_id'];
+    } else {
+	    $tillid = 'FFFFFF';
```

Comment:
I'm not sure it is a good idea to have fixed maximum value for ID.
Why not generating an other filter in this case?